__tests__/sources.ts

```diff
@@ -105,16 +160,45 @@ beforeEach(async () => {
 
 afterAll(() => disposeGraphQLTesting(state));
 
+describe('query sourceCategories', () => {
+  it('should return source categories', async () => {
+    const res = await client.query(`
+      query {
+        sourceCategories {
+          id
+          value
+          enabled
+          createdAt
+        }
+      }
+    `);
+    expect(res.errors).toBeFalsy();
+    const categories = getSourceCategories();
+    expect(res.data.sourceCategories).toMatchSnapshot(
```

Comment:
Can we maybe avoid snapshot and use something like this? https://jestjs.io/docs/expect#expectarraycontainingarray
Or simply toEqual?

Reply:
Sure, let's do that.